list.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+  
+@Pipe({
+  name: 'list',
+})
+export class ListPipe implements PipeTransform {
+  result: any;
+  transform(array: any): any {
+    this.result = array.join(' ');
```

Comment:
You no need ti use class property

you can use:
const result = array.join(' ');
or even:
return array.join(' ');